test/data/css/tests.js

```diff
@@ -209,6 +210,36 @@ exports.test_data = {
             }
         ]
     }, {
+        name: "Preserve Newlines",
+        description: "",
+        matrix: [{
+            options: [
+                { name: "preserve_newlines", value: "true" }
+            ],
+            separator_input: '\\n\\n',
+            separator_output: '\\n\\n',
+            separator_output_tabs: '\\n\\t\\n\\t',
```

Comment:
Nope, sorry.  
You've changed the tests to match the output when the output was wrong. 
There should be no tabs on blank lines.